tests/gp/test_hsgp_approx.py

```diff
@@ -108,12 +108,12 @@ def model(self):
 class TestHSGP(_BaseFixtures):
     def test_set_boundaries_1d(self, X1):
         X1s = X1 - np.mean(X1, axis=0)
-        L = pm.gp.hsgp_approx.set_boundary(X1s, c=2).eval()
+        L = pm.gp.hsgp_approx.set_boundary(X1s, c=2)
```

Comment:
Can we parametrize the test to make sure is cover the cases when `Xs` is always negative and when it contains negative values and positive ones?

Reply:
Done